content/articles/qnap-monitoring-environment.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Using SNMP to monitor a QNAP device"
+date: 2020-01-10
+type: "post"
+authors: ["anirudh"]
+tags: ["SNMP","monitoring","grafana","collectd"]
+lede: "It’s very useful to have all our monitoring in a central environment.
+We use Graphite and Grafana to collect metrics from our linux servers, and
+here’s how we added our QNAP file server using SNMP and collectd."
+shortlede: ""
+poster: "poster-qnap-monitoring-environment.jpg"
+thumbnail: "poster-qnap-monitoring-environment.jpg"
+socmediaimg: "socmediaimg-qnap-monitoring-environment.jpg"
+hiliteimg: "poster-qnap-monitoring-environment.jpg"
+poster_sourceurl: "https://www.pinclipart.com/pindetail/Txhmbi_office-management-clipart-performance-monitoring-png-download/"
+poster_source: "Pinclipart.com"
+bookendanimal: "satellite-dish"
+---
+
+At the CTL, we use Graphite and Grafana to monitor all of our server
+infrastructure, but our last physical server, a QNAP file server, does not have
+a straightforward way to send its metrics to a graphite monitoring
+environment.  Even worse, it runs a customized version of Linux which we were
+hesitant to modify.  Luckily it does support the Simple Network Management
+Protocol (SNMP), which can do just that.
+
+## What is SNMP?
+
+Essentially it is a network protocol for collecting information about managed
```

Comment:
"Essentially" seems unnecessary here. I would just say, "It is a network protocol for collecting information about managed devices."